containers/images/pulp/container-assets/route_paths.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import pkgutil
+import json
+import re
+import sys
+import nginx
+from pathlib import Path
+
+nginx_configs = []
+
+for i in pkgutil.iter_modules():
+    # filter pulp and galaxy packages
+    if re.search(r'pulp_.*|galaxy_ng',i.name):
+        # only add the filename to array if it exists
+        if Path(i.module_finder.path + "/" + i.name + "/app/webserver_snippets/nginx.conf").is_file():
+            nginx_configs.append(i.module_finder.path + "/" + i.name + "/app/webserver_snippets/nginx.conf")
+
+name = sys.argv[1]
+router = []
+
+for plugin_conf in nginx_configs:
```

Comment:
does it makes sense to raise an error in case this list happens to be empty?

Reply:
if someone uses a custom image with a plugin that doesn't have nginx snippet, it would break them

Reply:
hum... not sure if it would be helpful because in case there is no plugin installed it will just return
an empty list that our ansible tasks will take and do nothing with it, whereas an exception would trigger
an ansible `fail` task execution.

Reply:
Having installed just pulpcore without any plugin is a useless installation per se :)
@fao89 you are right, not all plugins have nginx snippets :imp: